core/src/main/java/io/confluent/rest/ApplicationServer.java

```diff
@@ -424,14 +429,17 @@ private void configureConnectors(SslContextFactory sslContextFactory) {
         SUPPORTED_URI_SCHEMES, "http");
 
     for (NamedURI listener : listeners) {
-      addConnectorForListener(httpConfiguration, httpConnectionFactory, listener, http2Enabled);
+      addConnectorForListener(httpConfiguration, httpConnectionFactory, listener,
+              http2Enabled, proxyProtocolEnabled);
```

Comment:
nit: Personally I prefer formatting where either all method parameters are on the same line or each method parameter is on its own line, but unlike `kafka-rest`, `rest-utils` doesn't have a well established formatting rule about that, so you don't necessarily need to change this.

Reply:
The existing practice in this file at least seems to be mostly multiple parameters per line, so I'm going to stick with that